libdevcore/CommonData.h

```diff
@@ -358,4 +358,10 @@ bool contains(T const& _t, V const& _v)
 	return std::end(_t) != std::find(std::begin(_t), std::end(_t), _v);
 }
 
+template <class V>
+bool contains(std::unordered_set<V> const& _set, V const& _v)
+{
+    return _set.find(_v) != _set.end();
```

Comment:
You can still use `.count()` here.

Reply:
I'm not a fan of this idiom, I think `find()` better expresses the intent.

Reply:
:+1: